### PR TITLE
Proposal: simplify auto import descriptions

### PR DESCRIPTION
**Pull request by andrewbranch**
Compliments #47552 and makes progress on https://github.com/microsoft/TypeScript/issues/47521.

Now, when a codefix will result in a new import declaration, the message is `Add import from "foo"`. When adding an import to an existing import declaration, the message is `Update import from "foo".` The exception is when the thing being imported doesn’t share a name with the identifier that triggered the error—the only way this happens today is when you are prompted to import `React` (or whatever your JSX element factory is) on an arbitrary JSX element name. In that case, the message is `Import 'React' from "react"`.

This change is based on two assumptions:

- You don’t need to see the name of the thing you’re importing repeated in the description because you’re fixing an error on that name or are in the process of typing that name.
- Nobody really cares whether the import is going to be a default import or named import.

### CODE REVIEW

File: src/compiler/diagnosticMessages.json [hunk 1]
```diff
@@ -6354,18 +6354,14 @@
         "category": "Message",
         "code": 90012
     },
-    "Import '{0}' from module \"{1}\"": {
+    "Import '{0}' from \"{1}\"": {
```
Comment (amcasey) [inline src/compiler/diagnosticMessages.json:6357]: This is used for both add and update?  Also, nitpicky, but why one set of single quotes and one set of double quotes?

**Review by mjbvz — APPROVED** [review]
The new wording makes sense to me. Thanks for taking a look!

**Review by amcasey — APPROVED** [review]

File: src/services/codefixes/importFixes.ts [hunk 2]
```diff
@@ -945,11 +954,9 @@ namespace ts.codefix {
                     importKind === ImportKind.Named ? [{ name: symbolName, addAsTypeOnly }] : emptyArray,
                     compilerOptions);
                 const moduleSpecifierWithoutQuotes = stripQuotes(moduleSpecifier);
-                return [
-                    importKind === ImportKind.Default ? Diagnostics.Add_default_import_0_to_existing_import_declaration_from_1 : Diagnostics.Add_0_to_existing_import_declaration_from_1,
-                    symbolName,
-                    moduleSpecifierWithoutQuotes
-                ]; // you too!
```
Comment (DanielRosenwasser) [inline src/services/codefixes/importFixes.ts:952]: you too!

File: src/compiler/diagnosticMessages.json (hunk 1 above)
Comment (DanielRosenwasser) [reply src/compiler/diagnosticMessages.json:6357]: I don't think I even see this message used the diff. Am I missing it?

Comment (amcasey) [reply src/compiler/diagnosticMessages.json:6357]: I missed it too - it's referred to by its property name in the tests.

Comment (andrewbranch) [reply src/compiler/diagnosticMessages.json:6357]: @amcasey yes, both add and update. We always use double quotes for module specifiers for some reason, and single quotes for other code quoting 🤷‍♂️

File: src/services/codefixes/importFixes.ts (hunk 2 above)
Comment (andrewbranch) [reply src/services/codefixes/importFixes.ts:952]: I left it in for so long, never knowing what it was referencing.

**Pull request merged**